raiden-dapp/package.json

```diff
@@ -58,7 +59,7 @@
     "@vue/cli-service": "^4.2.2",
     "@vue/eslint-config-prettier": "^6.0.0",
     "@vue/eslint-config-typescript": "^5.0.2",
-    "@vue/test-utils": "^1.0.0-beta.31",
+    "@vue/test-utils": "1.0.0-beta.31",
```

Comment:
Pinned it because otherwise `npm run build` for `raiden-dapp` fails, see https://github.com/raiden-network/light-client/pull/1205